Toshi/Source/Toshi/File/TTSF.h

```diff
@@ -1,9 +1,44 @@
 #pragma once
+#include "Toshi/File/TFile.h"
+
 namespace Toshi
 {
 	class TTSF
 	{
-		int m_iReadPos; // 0x120
+	public:
+		enum class Endianess
+		{
+			Little,
+			Big,
+		};
+
+		struct Header
+		{
+			uint32_t m_Magic;
+			uint32_t m_FileSize;
+		};
+
+		// idk how to call it and what's it's purpose
+		struct FileInfo
+		{
+			uint32_t FileStartOffset; // offset to TRBF
+			uint32_t FileSize;        // just the size
+		};
+
+	public:
+		uint8_t ReadFile(TFile* a_pFile);
```

Comment:
I was about to tell you that i found out that the function needs to be in ttsf, but you figured it out yourself